p5/sketch/Skia2DRenderer/renderer2d.py

```diff
@@ -29,6 +29,17 @@ class Style2D:
     stroke_cap = skia.Paint.kRound_Cap
     stroke_join = skia.Paint.kMiter_Join
 
+    # Flag that holds whether stroke() is called user
+    stroke_set = False
```

Comment:
Can we eliminate the `stroke_set` and always use the default values if the user hasn't set the stroke style?

Reply:
Texts are drawn a bit differently. By default, they are drawn such that there is no outline or stroke only fill is used. We should not stroke the text until the user has explicitly called `stroke()`, hence the variable `stroke_set` is used. 

Without it, all texts drawn would have an outline as well, which is different from the output of p5.js

Reply:
I see. I think there's also the `stroke_enabled` variable though. Can we use that to indicate whether the user has called `stroke`? (We'll have to set it to false when user calls `no_stroke`, but I think that's fine).

Reply:
I am not sure if that would be possible. Things we have to take care of are 
- After each frame these variables are to be reset, so if a sketch ends with `no_stroke()` It should not affect the next draw calls, so we set `stroke_enabled` to `True`
- Text on the other hand is drawn with stroke disabled by default. To get a stroke on the text we have to call stroke() explicitly.
- A single variable would never solve the ambiguity of what must have happened in the sketch for `text` and `shape` APIs. Let's say we have `stroke_enabled` to True by default. Then `text` APIs should somehow know that the current `True` value of `stroke_enabled` is not something we got after explicitly calling `stroke`. 
Similarly, if we would have `stroke_enabled` to `False`, then `shape` APIs would not know whether the `no_stroke` was called or we using the default value. 

So, for that, we would need a `stroke_set` variable.

Reply:
Thanks for the explanation! I now understand that we need to variables because `stroke` is default on for shapes and default off for fonts. I do wonder if there are better names for these variables, but this is good for now.